### PR TITLE
fix typos in docs

### PR DESCRIPTION
**Pull request by garanews**

### CODE REVIEW

**Review by adamgordonbell — APPROVED** [review]
Thanks @garanews! 
If you'd like some earthly stickers checkout this blog post: https://earthly.dev/blog/hacktoberfest-2021/

File: docs/guides/registries/self-signed.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ global:
       http = true|false
       insecure = true|false
       ca=["<ca-path-pem>"]
-      [[registry."<registry-hostname>".keypair]]
+      [[registry."<registry-hostname>".key pair]]
```
Comment (alexcb) [inline docs/guides/registries/self-signed.md:71]: This shouldn't be changed as it introduces whitespace.

**Review by alexcb — CHANGES_REQUESTED** [review]
thanks for the fixes, just one thing should be left as-is.

File: docs/guides/registries/self-signed.md (hunk 1 above)
Comment (garanews) [reply docs/guides/registries/self-signed.md]: Oh sorry, didn't notice it was a field.

**Pull request merged**